eZ/Bundle/EzPublishCoreBundle/Translation/Collector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the Collector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Translation;
+
+/**
+ * Retrieves all installed ezplatform translation files.
```

Comment:
Maybe specify that it looks for them within matches of "vendor/ezsystems/ezplatform-i18n-*" ?